services/frontend/src/components/UpdateNews.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-form
+    ref="form"
+    v-model="valid"
+    @submit="
+      (e) => {
+        e.preventDefault();
+        updateNews();
+      }
+    "
+  >
+    <div v-if="addingNewsDone">
+      <h1>Aktualizacja przebiegła pomyślnie</h1>
+      <v-divider />
+    </div>
+    <h1 v-if="addingNewsError">Problem ze zmianą aktualności</h1>
+    <h2 v-else>Zmiany w aktualności</h2>
+    <v-text-field v-model="title" :rules="rules.required" label="Tytuł" />
+    <v-textarea
+      solo
+      v-model="description"
+      :rules="rules.required"
+      label="Opis"
+    />
+    <v-btn v-bind:disabled="!valid" color="success" class="mr-4" type="submit">
+      Aktualizuj
+    </v-btn>
+  </v-form>
+</template>
+
+<script>
+import { api } from "@/api";
+
+export default {
+  name: "UpdateNewsForm",
+  props: ["newsId"],
+  data() {
+    return {
+      title: null,
+      description: null,
+      addingNewsDone: false,
+      addingNewsError: false,
+      valid: false,
+      rules: {
+        required: [(v) => !!v || "Wymagane pole"],
+      },
+    };
+  },
+  beforeMount() {
+    this.$store.commit("resetRegistration");
+    this.getSingleNewsData();
+  },
+  methods: {
+    async getSingleNewsData() {
+      try {
+        const news = await api.getNewsById(this.newsId);
+        this.title = news.data.title;
+        this.description = news.data.description;
+      } catch (e) {
+        this.$store.dispatch("actionCheckApiError", e);
```

Comment:
If you doing that, there should be check if user is still logged in, and redirect if isn't. I think that this is not needed, but i'd leave it